From a90acf38464d716b4506da3b839dfc59052817ba Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Mon, 25 Dec 2017 16:36:30 +0800
Subject: [PATCH 3280/5242] MLK-17289-8: dts: Remove hdmi_cec property

commit  31c35df192d73dbaf67abb2bfdf874bcf9fef9f9 from
https://source.codeaurora.org/external/imx/linux-imx.git

hdmi cec function is implement hdmi driver.
so remove hdmi_cec property in imx8qm dts files.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mq-evk-dcss-adv7535.dts  |    4 ----
 .../dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts  |    4 ----
 .../dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts |    4 ----
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi      |    8 +-------
 4 files changed, 1 insertion(+), 19 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-adv7535.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-adv7535.dts
index 0bc23ea..360bbb3 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-adv7535.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-adv7535.dts
@@ -22,10 +22,6 @@
 	status = "disabled";
 };
 
-&hdmi_cec {
-	status = "disabled";
-};
-
 &dcss {
 	status = "okay";
 	disp-dev = "mipi_disp";
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
index 804a6c8..c537a03 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
@@ -18,10 +18,6 @@
 	status = "disabled";
 };
 
-&hdmi_cec {
-	status = "disabled";
-};
-
 &dcss {
 	status = "okay";
 	disp-dev = "mipi_disp";
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts
index 36069c3..b45f6be 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-lcdif-adv7535.dts
@@ -36,10 +36,6 @@
 	status = "disabled";
 };
 
-&hdmi_cec {
-	status = "disabled";
-};
-
 &lcdif {
 	status = "okay";
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 1a09e3a..802d638 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -567,7 +567,7 @@
 		#address-cells = <1>;
 		#size-cells = <0>;
 		compatible = "fsl,imx8mq-hdmi";
-		reg = <0x0 0x32c00000 0x0 0x33800>,	/* HDP registers */
+		reg = <0x0 0x32c00000 0x0 0x100000>,	/* HDP registers */
 				<0x0 0x32e40000 0x0 0x40000>;   /* HDP SEC register */
 		interrupts = <GIC_SPI 16 IRQ_TYPE_LEVEL_HIGH>,
 					<GIC_SPI 25 IRQ_TYPE_LEVEL_HIGH>;
@@ -1228,12 +1228,6 @@
 		only-dma-mask32 = <1>;
 	};
 
-	hdmi_cec: hdmi_cec@32c33800 {
-		compatible = "fsl,imx8-hdp-cec";
-		reg = <0x0 0x32c33800 0x0 0x200>;
-		status = "disabled";
-	};
-
 	pcie0: pcie@0x33800000 {
 		compatible = "fsl,imx8mq-pcie", "snps,dw-pcie";
 		reg = <0x0 0x33800000 0x0 0x400000>, <0x0 0x1ff00000 0x0 0x80000>;
-- 
1.7.9.5

